src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-virtual-wan/main.bicep

```diff
@@ -0,0 +1,6 @@
+﻿// $1 = virtualWan
+// $2 = 'name'
+// $3 = Standard
```

Comment:
I'll be adding another standard to the docs shortly: "Inside quotes should be used on placeholder strings". Can you please add quotes to 'Standard'?